docs/api/target/class_balance.rst

```diff
@@ -7,6 +7,14 @@ One of the biggest challenges for classification models is an imbalance of class
 
 There are several techniques for dealing with class imbalance such as stratified sampling, down sampling the majority class, weighting, etc. But before these actions can be taken, it is important to understand what the class balance is in the training data. The ``ClassBalance`` visualizer supports this by creating a bar chart of the *support* for each class, that is the frequency of the classes' representation in the dataset.
 
+=================   ==============================
+Visualizer           :class:`~yellowbrick.target.class_balance.ClassBalance`
+Quick Method         :func:`~yellowbrick.target.class_balance.class_balance`
+Models               Classification
+Workflow             Feature analysis, Target analysis, Model selection
```

Comment:
Nice! I think we're pretty together a very good taxonomy of ML workflow that might be significant on its own.